### ISSUE TITLE
Rust panic on wrangler publish - Trying to publish tutorial(s) to workers.dev 

### ISSUE THREAD

**Issue by mgrollins** [cloudflare/wrangler#390]
When attempting to `wrangler publish` to my confirmed workers.dev domain, I get the below error about the main thread panicking. Below is the the output after `set RUST_BACKTRACE=1` in a normal user Windows command terminal.

`wrangler preview` command **did** work and launch preview as expected. I am running Node v10.11.0, but am unsure of what other environment details would help resolve this issue. Please let me know and I will update!

I received the below error output on the publish command from the quick start and qr tutorial: 

```
C:\Users\matt\Documents\Code\webworkers\qr-code-generator>wrangler publish
⬇️ Installing wranglerjs...
⬇️ Installing wasm-pack...
 Built successfully, built project size is 44 KiB.
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: InvalidHeaderValue { _priv: () }', src\libcore\result.rs:999:5
stack backtrace:
   0: <unknown>
   1: <unknown>
   2: <unknown>
   3: <unknown>
   4: <unknown>
   5: <unknown>
   6: <unknown>
   7: <unknown>
   8: <unknown>
   9: <unknown>
  10: <unknown>
  11: <unknown>
  12: <unknown>
  13: <unknown>
  14: <unknown>
  15: <unknown>
  16: <unknown>
  17: <unknown>
  18: BaseThreadInitThunk
  19: RtlUserThreadStart
```

**Comment by signalnerve** [comment]
Hey @mgrollins - interesting! I haven't heard of anyone running into problems with the QR code generator tutorial (assuming you mean [this one](https://workers.cloudflare.com/docs/tutorials/build-a-serverless-function/)), so would definitely like to help you get this working!

Can you fill this out? Will make it easier for us to debug:

```
#### What version of Wrangler are you using (`wrangler -V`)?

<pre>
$ wrangler -V

</pre>

#### Does this issue reproduce with the latest release?



#### What environment are you using?
* operating system:
* rustc version:
* node version:
```

**Comment by xortive** [comment]
This is almost certainly a duplicate of #389, just reproduced on a windows machine.

**Comment by ashleygwilliams** [comment]
closing this as a duplicate! we will fix this for a point release (1.1.1) that should be out by the end of the week. you can track the fix at #389
